projects/laji/src/app/+project-form/results/syke-insect-result/syke-insect-route-table/syke-insect-route-table.component.ts

```diff
@@ -13,10 +13,11 @@ import { DateFormatPipe } from 'ngx-moment';
   styleUrls: ['./syke-insect-route-table.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class SykeInsectRouteTableComponent {
+export class SykeInsectRouteTableComponent implements OnInit, OnChanges {
   @Input() routeId: string;
   @Input() season: string;
   @Input() sorts: {prop: string, dir: 'asc'|'desc'}[] = [];
+  @Input() activeYear;
```

Comment:
This is missing a type. The taxonSet variable below seems to also be missing it

Reply:
Fixed by removing both, they were remnants from testing and should not have been left there.

Reply:
If I'm looking at correct version, taxonSet is still used in the `syke-insect-route-table.component.html` template (2nd line)

Reply:
Right you are, I missed that use, returning it and setting it's type which should be the right one.

Reply:
:+1: